source/includes/stackpath/_sites.md

```diff
@@ -961,3 +961,36 @@ Optional | &nbsp;
 `backupOrigin.username` <br/> *string* | Username to use when authenticating with the backup origin. 
 `backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
 `backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+
+<!-------------------- RETRIEVE EDGESSL SETTINGS -------------------->
+
+### Retrieve EdgeSSL settings
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/edgesslsettings/eb3ecdbe-d73b-40e6-a263-166accba75ed"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": "eb3ecdbe-d73b-40e6-a263-166accba75ed",
+    "forceHttps": true,
```

Comment:
Either put this attribute in quotas or mark the block with `js` instead of `json`.